### DISCUSSION TITLE
Qual faculdade para defesa cibernética ?

### DISCUSSION THREAD

**Discussion by JhonnyO3** [frontendbr/forum#2234] [Perguntas e Respostas]
### Discussed in https://github.com/frontendbr/forum/discussions/809

<div type='discussions-op-text'>

<sup>Originally posted by **Lucas1906** November 13, 2017</sup>
Pessoal decidi q irei fazer um curso superior...  Mesmo analisando e vendo q não é algo tao necessário para área de front end algumas coisas fizeram eu decidir em fazer... Quero algo rapido entao irei fazer um tecnólogo. Quais faculdades vcs indicariam ? andei vendo 3, FIAP, São Judas, Anhebi.  Ja q é um curso rapido to levando em consideração o nome delas no mercado e a nota no Mec. Se alguem estuda/etuou em algumas delas poderia deixa uma opinião. Obs: to quase decidindo pela FIAP pela localização e pelo nome </div>

**Comment by JailtonDomingos** [comment]
Olá Dev, não responderei sua pergunta mas deixarei um conhecimento a mais para sua jornada.

Você não precisa necessariamente ter uma graduação, porém certificações com peso mundial são extremamente necessárias para profissionais em Cyber Security que se preze. Segue uma listinha das destaques:

> CISSP – CERTIFIED INFORMATION SYSTEMS SECURITY PROFESSIONAL – DO ISC2
> CEH – CERTIFIED ETHICAL HACKER – DO EC-COUNCIL
> OSCP – OFFENSIVE  CERTIFIED SECURITY PROFESSIONAL – PELA OFFENSIVE SECURITY 
> CISM – CERTIFIED INFORMATION SECURITY MANAGER
> COMPTIA SECURITY +

Observação: Eu pessoalmente acredito que o trabalho com Big Data e Inteligência analítica esteja extremamente ligado com esta área, por isso estou em formação neste campo inicial para depois procurar especializações(Pós, Mestrado e doutorado) ou até mesmo as certificações na área de Ciber Security.

** Desculpe não responder sua pergunta, mas acredito que conhecimento nunca é demais. Caso você já saiba disto, por favor ignore!

Bom code!
